src/guides/v2.3/config-guide/cache/two-level-cache.md

```diff
@@ -0,0 +1,62 @@
+---
+group: configuration-guide
+title: Using TwoLevel cache in Magento application
+functional_areas:
+  - Configuration
+  - System
+  - Cache
+---
+
+## Starting from 2.3.5 there is a possibility to set up TwoLevel Syncronized cache as base cache adapter.
+
+###Overview.
+The main idea is to reduce network size transferring between remote cache storage and application. In Vanilla Magento instance we transfer around 300kb per 1 request and may reach network limitation, which is mainly up to 10Gbps, in around of ~1000 requests.
+
+To overcame this and reduce pressure on redis we keep cache data on each web node locally and use Remote Cache for two purposes:
+- Check only data version to ensure that we have latest cache locally.
+- Transfer latest cache from remote to local if we have stale one.
+
+We keep hashed data version in remote in addition to data, it has suffix ':version' in addition to ordinary key. In case we have outdated data locally we will transfer it form remote and save with a local cache adapter.
+
+###Configuration example. 
+
+```php?start_inline=1
```

Comment:
I think we can remove a redundant part - `?start_inline=1` and leave a php only

```
    ```php
```

Reply:
Sure, but if you do so you will have syntax error highlighted in your IDE because its not valid PHP code.

Reply:
What I can answer? :) We have a standard markup for code blocks. Ideally all code blocks should have the same markup. :) So, I've provided my suggestions regarding to this point.

Reply:
It's ok to use `?start_inline=1` here, no changes required

Reply:
If a code snippet is supposed to be a part of PHP file and it does not start with a PHP tag `<?php`, the code block should contain the `?start_inline=1` option to be processed by code highlighter as PHP code.